examples/scripts/delete_metric_descriptors/delete_metric_descriptors.go

```diff
@@ -0,0 +1,188 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+	"unicode"
+
+	monitoring "cloud.google.com/go/monitoring/apiv3/v2"
+	"cloud.google.com/go/monitoring/apiv3/v2/monitoringpb"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+)
+
+/*
+This script deletes metric descriptors from the given projects (-projects flag),
```

Comment:
Generally yes, but for one-file script it might be more accessible here? If we focus to always have one file for simplicity. It's robust code, but only an example after all.

Reply:
A `README.md` is more accessible because it's what users would naturally look for and we'd get nice rendering in GitHub, etc. There's some intimidation looking at the code but maybe for a sensitive script like this, that's what we want so I'm indifferent.